pkg/track/events.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package track
+
+import (
+	"context"
+	"sync"
+
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	"github.com/microsoft/retina/pkg/enricher"
+	"github.com/microsoft/retina/pkg/log"
+)
+
+type Track struct {
```

Comment:
Can you doc what a `Track` is intended to be?

Reply:
+1